README.md

```diff
@@ -30,8 +30,11 @@ func main() {
 	// Add your own RPC URL here or use a public one
 	rpcUrl := "https://rpc-mumbai.maticvigil.com"
 
+	// Instantiate a new provider to pass into the SDK
+	provider, _ := ethclient.Dial(chainRpcUrl)
```

Comment:
can we just do this under the hood? seems unnecessary for ppl to do this everytime